src/store/account/utils.test.ts

```diff
@@ -317,4 +323,144 @@ describe('utils', () => {
 			);
 		});
 	});
+
+	describe('updateIdentities', () => {
+		it('should return undefined if the account field in the state is undefined', () => {
+			setupAccountStore();
+			useAccountStore.setState((state) => ({
+				...state,
+				account: undefined
+			}));
+			const state = useAccountStore.getState();
+
+			expect(updateIdentities(state, {}, [])).toEqual(undefined);
+		});
+
+		it('should return the original identities if no mods are passed', () => {
+			setupAccountStore();
+			const state = useAccountStore.getState();
+			expect(updateIdentities(state, {}, [])).toEqual(state.account?.identities.identity);
+		});
+
+		it('should return the original identities and the new identities', () => {
+			setupAccountStore();
+			const state = useAccountStore.getState();
+			const newIdentity: Identity = {
+				_attrs: {},
+				id: faker.string.uuid()
+			};
+			expect(updateIdentities(state, {}, [newIdentity])).toEqual(
+				expect.arrayContaining([...(state.account?.identities.identity ?? []), newIdentity])
+			);
+		});
+
+		it('should return the original identities without the one which has to be deleted', () => {
+			setupAccountStore();
+			const state = useAccountStore.getState();
+			const identityToDeleteId = state.account?.identities.identity[0].id;
+			if (!identityToDeleteId) {
+				throw new Error('Identity not found');
+			}
+
+			const mods: IdentityMods = {
+				deleteList: [identityToDeleteId]
+			};
+			expect(updateIdentities(state, mods, [])).toEqual(
+				expect.not.arrayContaining([expect.objectContaining({ id: identityToDeleteId })])
+			);
```

Comment:
Why not using not.toContain instead of toEqual.not.arrayContaining? not.toContain seems more readable to me
```suggestion
			expect(updateIdentities(state, mods, [])).not.toContain(
				expect.objectContaining({ id: identityToDeleteId })
			);
```